src/components/Header/SocialLinks.tsx

```diff
@@ -2,74 +2,101 @@ import OrgIcon from "../Icons/OrgIcon";
 import ReposIcon from "../Icons/ReposIcon";
 import { TitleAndSocialLinkProps } from "./types";
 
-
-export default function SocialLinks({ setView, view }: TitleAndSocialLinkProps) {
+export default function SocialLinks({
```

Comment:
can you try to add the changes you entered without the formatting changes?

there are too many changes in this file.